openfecwebapp/templates/landing.html

```diff
@@ -0,0 +1,110 @@
+{% extends "layouts/main.html" %}
+{% import 'macros/cycle-select.html' as select %}
+{% import 'macros/overviews.html' as overviews %}
+{% set hide_header_search = true %}
+
+{% block title %}
+  {{ title }}
+{% endblock %}
+
+{% block body %}
+<section class="hero hero--primary hero--data" aria-labelledby="hero-heading">
+  <div class="hero__image"></div>
+  <div class="container">
+    <h1 id="hero-heading" class="hero__title">Campaign finance data</h1>
+    <p class="hero__content">See how candidates and committees raise and spend money in federal elections. This financial data helps voters make informed decisions.</p>
+  </div>
+</section>
+
+<section class="slab slab--neutral slab--spacious">
+  <div class="container">
+    {{ search.search('hero', button_color="button--standard", select_class="select--alt") }}
+    <span class="t-note t-sans search__example">Examples: George Bush; Obama for America; C045378</span>
+  </div>
+</section>
+
+<div class="main container js-accordion accordion--neutral" data-content-prefix="about"">
+  <section class="content__section--ruled content__section--extra">
+    <div class="grid grid--2-wide">
+      <div class="grid__item grid__item--no-margin card card--wide">
+        <div class="card__image__container">
+          <img class="card__image icon--complex" src="{{ url_for('static', filename='img/i-elections--neutral.svg') }}"  alt="Icon representing elections">
+          <h2 class="card__title"><a href="{{ url_for('election_lookup') }}">Elections by location &raquo;</a></h2>
+        </div>
+        <div class="card__content">
+          <h2 class="card__title"><a href="{{ url_for('election_lookup') }}">Elections by location &raquo;</a></h2>
+          <span>Find elections near you— search by state or ZIP code.</span>
```

Comment:
I don't think there is supposed to be space after the dash. cc @emileighoutlaw ?